backend/apps/slack/admin/__init__.py

```
@@ -1,7 +1,8 @@
 """Slack app admin."""
-
+ 
+from .bot_interaction import BotInteractionAdmin  # ← ADD THIS LINE
 from .conversation import ConversationAdmin
 from .event import EventAdmin
 from .member import MemberAdmin
 from .message import MessageAdmin
-from .workspace import WorkspaceAdmin
+from .workspace import WorkspaceAdmin
```

Ruff (0.15.18)

[warning] No newline at end of file

Add trailing newline

(W292)